bin/citgm-all.js

```diff
@@ -97,6 +119,10 @@ function runCitgm (mod, name, next) {
   const runner = citgm.Tester(name, options);
   let bailed = false;
 
+  if (checkTags(app, mod, name)) {
+    return next(); // Skip this module
```

Comment:
we are generally at the moment, let's discuss elsewhere and find a linter rule once this is landed